src/app.service.ts

```diff
@@ -3,6 +3,6 @@ import { Injectable } from '@nestjs/common';
 @Injectable()
 export class AppService {
   getHello(): string {
-    return 'Hello World!';
+    return 'Hello from Auth!';
```

Comment:
To nie jest zmiana związana z MongoDB i jego konfiguracją.

Reply:
Masz rację. Sprawdzałam tutaj jak działa Nest xD. Wracam do Hello World.